pizzaplace/services/pizza_place_presenter.py

```diff
@@ -0,0 +1,19 @@
+class PizzaPlacePresenter(): 
+  def __init__(self, pizza_place, YelpApi):
+    self.pizza_place = pizza_place
+    self.yelp_api = YelpApi
```

Comment:
The capitalization in this variable name is a little odd, though. Even if the thing passed is a class, it's typical to give it a lowercase variable name, like `yelp_api`.